src/main/java/team4/footwithme/team/service/TeamServiceImpl.java

```diff
@@ -60,8 +57,9 @@ public TeamCreatedResponse createTeam(TeamCreateServiceRequest dto) {
         return TeamCreatedResponse.of(createdTeam);
     }
 
-    // 한 서비스에서 다 들고오는게 좋은건지 각각의 서비스로 운용해야되는건지 모르겠음
+
     @Override
+    @Transactional
```

Comment:
getTeamInfo에 `@Transactional` 추가 하신걸로 보이는데 메서드이름만으로 유추해보았을 때는 팀 정보를 조회하는걸로 보여서 readOnly를 사용하는게 좋아보입니다.
```suggestion
    @Transactional(readOnly = true)
```

조회용 메서드일 경우에는 `@Transactional`을 붙일 필요가 없는데 저는 조회용메서드임을 표현하기 위해 명시적으로 작성합니다

Reply:
아 감사합니다!!